tests/environment.py

```diff
@@ -249,9 +252,9 @@ class ALEntityDocker(ALEntity):
 
     The entity is defined from the name of the container, the rest is derived from that.
     '''
-    # NOTE: name arg can be also extracted from the device class itself, but test_flows.py
-    # don't have it. We can remove this arg as soon, as we drop test_flows.py
-    def __init__(self, name: str, device: None = None, is_controller: bool = False):
+    def __init__(self, name: str, device: None = None, is_controller: bool = False,
```

Comment:
I'm not sure why the comment was removed, test_flows.py still needs it AFAIK.

Reply:
Fixed on c34ceebc